rules/windows/malware/av_spearphishing.yml

```diff
@@ -0,0 +1,33 @@
+title: Antivirus Relevant File Name Alerts for Spearphishing
+id: 978831de-6bf6-4b9d-b26a-c29f04d7993b
+description: Detects spear phishing by antivirus alerts with strings an attacker might choose for the file name of his malicious attachment e.g. a providers name for a spoofed invoice. For example a car maker will get lots of common malware with DHL, UPS and Amazon in the file name but if the name of a clutch supplier is used, it might be spear phishing. This is not the ususal sigma rule where Flo or Thomas have already done all the work and you can lean back. You need to fill in the relevant strings an attacker might use for your specific organisation in the Filename section. Usually that will be providers, suppliers, big customers or any other organsiation with which you often exchange attachments. A convinient thing is that you get hits not only from the file names the attacker choose, but also from the directories your users sort the files into. For example, the filename might be only Invoice_20190411.doc, but if your user sorts it into \work\provider1\, you also get the alert (Well, if it's not immediatly caught in \outlook\temp and your AV finds it after the latest signature update in 2 weeks). If you experience too many false positives, you might want to restrict the alerts to AV signatures used by malicious attachments like e.g. "Downloader" and "Exploit" (depending of course on how your AV vendor calls it). Apart from that, the amount of false positives depends of course on how many entries you make and how short and common those strings are. Happy tweaking.
+date: 2020/02/12
+modified: 20/02/12
+author: Arnim Rupp
+tags:
+        attack.t1192
+        attack.t1193
+        attack.t1194
+logsource:
+    product: antivirus
+status: experimental
+detection:
+    selection:
+        FileName:
+            - 'organisation1'
+            - 'organisation2'
+            - 'provider1'
+            - 'provider2'
+            - 'customer1'
+            - 'customer2'
+            - 'supplier1'
+            - 'supplier2'
+            - 'your own organisations name? might be useful but will also throw lots of false positives ...'
+            - 'your own organisations abriviatet name? even more false positives ...'
```

Comment:
This is more a template than a rule itself. Using the rule as-is will likely not generate lots of true positives.